src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java

```diff
@@ -18,21 +18,20 @@
 
 @Service
 public class PaymentsProcessor {
-    private static final Logger LOG = LoggerFactory.getLogger(PaymentsProcessor.class);
+
+    private static final Logger log = LoggerFactory.getLogger(PaymentsProcessor.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The constant name 'log' doesn't match '[A-Z][A-Z_0-9]*'](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4543433)